network/802.11/hccap.ksy

```diff
@@ -0,0 +1,57 @@
+meta:
+  id: hccap
+  title: Hashcat capture old format
+  license: Unlicense
+  file-extension: hccap
+  application:
+    - Hashcat
+    - aircrack-ng
+  endian: le
+  encoding: utf-8
+  imports:
+    - /network/mac_address
+    - /network/802.11/eapol_key
+    - /network/802.11/wpa2_nonce
+doc: |
+  Native format of Hashcat password "recovery" utility
+  A sample of file for testing can be downloaded from https://web.archive.org/web/20150220013635if_/http://hashcat.net:80/misc/example_hashes/hashcat.hccap
+doc-ref: https://hashcat.net/wiki/doku.php?id=hccap
+seq:
+  - id: records
+    type: hccap
+    repeat: eos
+types:
+  hccap:
+    seq:
+      - id: essid
+        type: strz
+        size: 36
```

Comment:
I *heavily* doubt that ESSID is an UTF-8 string. AFAIR, lots of devices use a very arbitrary choice of encoding, so I guess this one should be treated as byte array.